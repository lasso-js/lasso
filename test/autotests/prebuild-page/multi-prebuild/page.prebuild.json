[
  {
    "slots": {
      "body": "<script src=\"/static/page/lasso$3.4.3/test/autotests/prebuild-page/multi-prebuild/a.js\"></script>"
    },
    "assets": [],
    "name": "page"
  }
]