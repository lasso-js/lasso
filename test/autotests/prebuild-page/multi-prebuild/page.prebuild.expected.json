[
  {
    "slots": {
      "body": [
        {
          "contentType": "js",
          "content": [
            {
              "inline": false,
              "code": {
                "src": "/static/page/lasso/test/autotests/prebuild-page/multi-prebuild/a.js"
              }
            }
          ]
        }
      ]
    },
    "assets": [],
    "name": "page"
  }
]